bin/main.py

```diff
@@ -311,10 +312,24 @@ def find_optimal_parameters():
          str(config['test_param']['maxInterval']),
          str(config['test_param']['defaultGas']), '1'])
 
+    run_file(
+        ['bash', _get_path(MONITOR_PATH)])
+
     result = find_optimal_parameters()
     print("Best result found: " + str(result))
     print('Aggregating all the workload reports')
     run_file(['python', _get_path(AGGREGATE_RESULTS_PATH)])
     exec_time = (time.time() - start_time)
     print("Execution time: " + str(exec_time))
     exit(0)
+
+
+
+def load_args():
```

Comment:
I think this is unfinished? I guess your idea is to add argument parser to enable monitoring or not.

Reply:
i would remove it for now. we can add the flag later :)